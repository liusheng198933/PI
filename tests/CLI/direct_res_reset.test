#!/bin/sh
./run_one_test.py $srcdir/testdata direct_res_reset stats.json
