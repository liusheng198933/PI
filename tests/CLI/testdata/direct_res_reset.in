counter_write ExactOne_counter NEXT_ENTRY bytes=388
direct_res_reset
table_add ExactOne 0 => actionA 0
counter_read ExactOne_counter 0
