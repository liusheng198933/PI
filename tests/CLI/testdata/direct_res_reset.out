????
Device assigned successfully.
Selecting device.
????
Dumping counter entry:
	Bytes: 388
Direct resource spec was stored.
????
????
Entry was successfully added with handle 0.
????
Dumping counter entry:
	Packets: 0
	Bytes: 0
????
